src/libutil/references.cc

```diff
@@ -12,7 +12,10 @@
 namespace nix {
 
 
-static size_t refLength = 32; /* characters */
+/**
+ * A Nix reference is 32 characters long
```

Comment:
From what I know a "Nix"(?) reference is a relation between store objects, so this still makes no sense.
Is this about the store path hash part?

OT: why isn't this in libstore if it's about store concepts?

Reply:
This happened in 3ebe1341abe1b0ad59bd4925517af18d9200f818 with my assistance. I got fooled by the types and thought this was store agnostic :(. I agree that given how the code is written today, it doesn't belong here.

Reply:
@roberth I say we merge this after perhaps rephrasing the above, and then I'll move it back. How does that sound?